R-packages/covidcast/R/covidcast.R

```diff
@@ -437,18 +437,21 @@ covidcast_signals <- function(data_source, signal,
 #'
 #' @export
 covidcast_meta <- function() {
-  meta <- .request(list(source='covidcast_meta', cached="true"))
+  meta <- .request(
+    list(source = "covidcast_meta",
+         format = "csv"))
```

Comment:
Are we purposefully removing caching behavior here?

Reply:
Hysterical Raisins™. Caching behavior used to be opt-in, and if you didn't request the cached version, it would take forever to load. All requests now get the cached version: https://github.com/cmu-delphi/delphi-epidata/commit/ed19bc4f77f40e8695c04a13a2d20a2072cfab47#diff-a9c5ce7767493542e52e74713cbbeacdee649c91a5c933f3b9981f5173826a3b

So that parameter was just wasting a few bytes.